content/resources/checklist-of-requirements-for-federal-digital-services.md

```diff
@@ -35,7 +35,16 @@ These high-level policies cover basic requirements for all websites and digital
 - See a full list of [policies and regulations »]({{< ref "/topics/policy" >}})
 - See our list of [**Required Web Content and Links »**](https://digital.gov/resources/required-web-content-and-links/?=checklist)
 
-<p class="more"><a href="{{< ref "/topics/21st-century-idea" >}}">More on 21st Century IDEA <i class="fas fa-arrow-alt-circle-right"></i></a></p>
+<p class="more">
+  <a href="{{< ref "/topics/21st-century-idea" >}}">More on 21st Century IDEA
+    <svg class="dg-icon dg-icon--standard margin-bottom-05" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+      <path d="M22 12c0-5.52-4.48-10-10-10S2 6.48 2 12s4.48 10 10 10 10-4.48 10-10zm-10 1H8v-2h4V8l4 4-4 4v-3z"/>
+    </svg>
+    <svg class="dg-icon dg-icon--standard margin-bottom-05" aria-hidden="true" focusable="false">
```

Comment:
Let's add the `usa-icon` class here if it's from the USWDS sprite.

Reply:
I have added a decorative class `circle-arrow-forward-icon` for the inline svg. And removed the second svg, it was a duplicate.